public/app/plugins/datasource/mssql/plugin.json

```diff
@@ -15,7 +15,8 @@
       "small": "img/sql_server_logo.svg",
       "large": "img/sql_server_logo.svg"
     },
-    "version": "%VERSION%"
+    "version": "%VERSION%",
```

Comment:
should we have added this version?

Reply:
it was already there - not added. or wdym?

Reply:
my bad!